main.js

```diff
@@ -0,0 +1,230 @@
+
+
+/* 
+/-----------------------------------------------------This is the Structure we want to build using DOM manipulation
+
+<li class="course5">
```

Comment:
you need to remove the comments from the final code

Reply:
I thought leaving the HTML structure commented above would facilitate reading and understanding the code